src/components/lab/ElectricFieldVectorGroup.js

```diff
@@ -11,36 +11,47 @@ import {
   SMALL_ARROW_Y_PLACEMENT_RELATIVE_TO_LARGE_ARROW,
 } from '../../config/constants';
 
-const ElectricFieldVectorGroup = ({ x, y, direction }) => {
+const ElectricFieldVectorGroup = ({ x, y, direction, absorptionPoint }) => {
   return (
     <Group>
-      <Arrow
-        x={x}
-        y={y - SMALL_ARROW_Y_PLACEMENT_RELATIVE_TO_LARGE_ARROW}
-        points={[0, 0, direction * SMALL_ARROW_LENGTH, 0]}
-        pointerLength={ARROW_POINTER_LENGTH}
-        pointerWidth={ARROW_POINTER_WIDTH}
-        stroke={ARROW_STROKE_COLOR}
-        strokeWidth={ARROW_STROKE_WIDTH}
-      />
-      <Arrow
-        x={x}
-        y={y}
-        points={[0, 0, direction * LARGE_ARROW_LENGTH, 0]}
-        pointerLength={ARROW_POINTER_LENGTH}
-        pointerWidth={ARROW_POINTER_WIDTH}
-        stroke={ARROW_STROKE_COLOR}
-        strokeWidth={ARROW_STROKE_WIDTH}
-      />
-      <Arrow
-        x={x}
-        y={y + SMALL_ARROW_Y_PLACEMENT_RELATIVE_TO_LARGE_ARROW}
-        points={[0, 0, direction * SMALL_ARROW_LENGTH, 0]}
-        pointerLength={ARROW_POINTER_LENGTH}
-        pointerWidth={ARROW_POINTER_WIDTH}
-        stroke={ARROW_STROKE_COLOR}
-        strokeWidth={ARROW_STROKE_WIDTH}
-      />
+      {/* only show the arrow if its y position is greater than the absorption point */}
+      {/* i.e. once the arrow is above the absorption point, don't show it */}
+      {/* (similar for the two other arrows below) */}
+      {y - SMALL_ARROW_Y_PLACEMENT_RELATIVE_TO_LARGE_ARROW >=
```

Comment:
Maybe factor out these inequalities to line 15 as `showFirstArrow`, `showSecondArrow`, etc.

Reply:
good call